modules/todo.js

```diff
@@ -1,9 +1,8 @@
-class Todo {
-  constructor(desc, completed, id) {
-    this.desc = desc;
+// Class Todo
+export default class Todo {
+  constructor(item, completed = false, index) {
+    this.item = item;
+    this.index = index;
     this.completed = completed;
-    this.id = id;
   }
```

Comment:
- [ Optional ] As stated the project requirement By default new tasks should have the property `index` set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5). But In your Project the property `index` doesn't setup the specific `id` Kindly Adjust it.
 
![Capture](https://user-images.githubusercontent.com/93598189/201476825-230c7551-47fd-49ee-917b-a7524166bd2f.PNG)